packages/kolme/src/core/types/error.rs

```diff
@@ -19,6 +42,306 @@ pub enum KolmeError {
         max_height: BlockHeight,
         proposed_height: BlockHeight,
     },
+
+    #[error("Unsupported wallet address: {address}")]
+    UnsupportedWalletAddress { address: String },
+
+    #[error("Invalid category name {name}")]
+    InvalidCategoryName { name: String },
```

Comment:
It doesn't look like this is used. Either way, the name and display could be clarified, I don't know what category means here.